GitUI/UserControls/RevisionGrid/RevisionGridControl.cs

```diff
@@ -162,7 +162,7 @@ private RefFilterOptions RefFilterOptions
         internal bool DoubleClickDoesNotOpenCommitInfo { get; set; }
 
         [CanBeNull]
-        internal ObjectId InitialObjectId { private get; set; }
+        internal ObjectId[] InitialObjectIds { private get; set; }
```

Comment:
Passing arrays around is inherently dangerous, do we expect more than 2 commits to be selected? If not, should we have two properties instead?

Reply:
More than two is an edge case, like if you want to see a submodule selected with the same commits you have multiselected or to select multiple commits from the cmd line. But that would be separate extensions.

Would passing a list around be OK?

Otherwise new arguments --selected-commit and --first-commit should be added (with backwards compatibility with -commit) should be added

Reply:
If automatically selecting more than two commits is an edge case I propose we have two `ObjectId` properties, thus removing the need to pass arrays around.
I propose we keep `InitialObjectId` as is (at least its functionality, but happy to have it renamed to something like `FirstSelectedCommit`), and add another property  for the second selected commit.

Reply:
I changed this even if I find that this makes the code harder to maintain.